test/persist-catalog-migration/mzcompose.py

```diff
@@ -0,0 +1,198 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from textwrap import dedent
+
+from materialize.mzcompose.composition import Composition
+from materialize.mzcompose.services.cockroach import Cockroach
+from materialize.mzcompose.services.materialized import Materialized
+from materialize.mzcompose.services.minio import Minio
+from materialize.mzcompose.services.testdrive import Testdrive
+
+SERVICES = [
+    Minio(setup_materialize=True),
+    Cockroach(setup_materialize=True),
```

Comment:
You don't seem to use external_minio and external_cockroach, so I think these services are not required.

Reply:
Thanks, removed.